parachain/src/service.rs

```diff
@@ -223,7 +223,8 @@ where
                     async move {
                         let slot_ms = match client_clone.clone().runtime_version_at(&BlockId::Hash(parent.clone())) {
                             Ok(x) if x.spec_name.starts_with("kintsugi") => 6000,
-                            _ => 12000,
+                            Ok(x) if x.spec_version >= 2 => 12000,
+                            _ => 6000,
```

Comment:
obviously this needs changing when we want to bring this for kintsugi - this was simplified for the testnet test